packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,266 @@
 <template>
   <VbDemo>
-    <VbCard>
-      <va-button @click="$refs.treeView.expand()">
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
```

Comment:
Please include icons here and anything else if it's supported in default tree view. It's not clear for me I can use icons without slot.